src/main/java/frc/robot/Constants.java

```diff
@@ -169,7 +169,7 @@ public AimModes previous() {
 
   // Intake Contstants
   public static final int intakeMotorOneID = 1;
-  public static final double intakeMotorSpeed = 0.70;
+  public static final double intakeMotorSpeed = 1.0;
```

Comment:
@Majorferguson21 did these come from testing -- intake and CDS wheel speeds? Want to make sure we're purposefully replacing these values and know they work.

Reply:
Yeah, that's the value we left off at